### ISSUE TITLE
IMG with SVG data URL gets broken on load

### ISSUE THREAD

**Issue by GabiAxel** [tinymce/tinymce#3864]
For example, after adding the following content, it is displayed and saved properly, but when reloading it, TinyMCE converts all images to blobs and somehow gets an empty URL for this image:

`
<img src="data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' height='100' width='100'%3E%3Ccircle cx='50' cy='50' r='40' stroke='black' stroke-width='3' fill='red' /%3E%3C/svg%3E"/>
`

TinyMCE 4.6.5

**Comment by kkuegler** [comment]
We are also seeing this with the latest TinyMCE 5.10.2.

We previously worked around this with this config:
```
images_dataimg_filter: function (img) {
	return img.src.indexOf('data:image/svg+xml') !== 0;
},
```
but as this is deprecated and will go away in 6.0, this bug will affect us again.

**Comment by kkuegler** [comment]
We got a new non-deprecated workaround:
```js
setup: function(editor) {
	editor.on('PreInit', function() {
		editor.editorUpload.addFilter(function(img) {
		  return img.src.indexOf('data:image/svg+xml') !== 0;
		});
	});
}
```
You can see this in action at https://fiddle.tiny.cloud/IRhaab/3

**Comment by kkuegler** [comment]
This is fixed in Tiny 6.1:
https://www.tiny.cloud/docs/tinymce/6/6.1-release-notes/#images-that-used-a-data-uri-were-corrupted-when-the-data-wasnt-base64-encoded

**Comment by TheSpyder** [comment]
Apologies, this issue was not linked to our tracking ticket when #7918 was implemented.
